core/arch/arm/pta/se_api_self_tests.c

```diff
@@ -77,30 +77,6 @@
  * Trusted Application Entry Points
  */
 
-static TEE_Result create_ta(void)
```

Comment:
Sorry, I thought I was commenting in `core/arch/arm/pta/pta_self_tests.c`. Just keeping it in `core/arch/arm/pta/pta_self_tests.c` should be enough.